### ISSUE TITLE
Unable to install pyhton3 env - support all installers installers (miniconda3, miniconda2 )

### ISSUE THREAD

**Issue by ieugen** [palantir/gradle-miniconda-plugin#20]
Hello,

I've used your plugin in anothr project and it worked great. It's the only sane way I found to build and package python apps across environments and team members. 

One issue I've noticed is that the release names used by miniconda has changed and I can't seem to figure out how to install miniconda3. 

I've checked the code and it seems to be related to how you choose the name of the installer. 

[1] https://repo.continuum.io/miniconda/

**Comment by mnazbro** [comment]
This doesn't seem too unreasonable. I'm in the middle of making some other cleanup changes, I can go ahead and add in the Miniconda 3 support.

**Comment by ieugen** [comment]
I made an internal copy (couldn't wait for a release) and ended up exposing `installerName` as an extension property and I eliminated all the logic from the plugin. It is much more robust that way. 

That and a simple documentation phrase sayng 'use Miniconda2 or Miniconda3 or whatever', plus a sane default if not present (I would go for python3 as python2 is being phased out).

p.s. Since my use case was running selenium and I failed to do that with python I ended up removing miniconda plugin from this new project so no need for me right now.
